examples/charm.scm

```diff
@@ -14,3 +14,4 @@
                        (rotate-y e (/ pi 2))))
 
 (ao-show model)
+(ao-export-mesh model "charm.obj" '(-1 -1 -1) '(1 1 1) 10)
```

Comment:
Can you remove this?  I'd rather not have sample files save meshes without any warning.